ngraph/core/include/ngraph/log.hpp

```diff
@@ -117,6 +117,10 @@ namespace ngraph
     NGRAPH_API
     void default_logger_handler_func(const std::string& s);
 
+#ifndef PROJECT_ROOT_DIR
+#define PROJECT_ROOT_DIR ""
```

Comment:
Why do you need to have this definition?

Reply:
PROJECT_ROOT_DIR is only ngraph specific definition so when we call NGRAPH_DEBUG not from ngraph we need this variable to be difined.